packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/runtimeIcon.st

```diff
@@ -0,0 +1,8 @@
+icons
+runtimeIcon
```

Comment:
If I understand the Squeak best-practices correctly, message names should include a verb. This leads to code reading more like an actual sentence. So please consider renaming this message to `getRuntimeIcon`.

One additional point:
`runtimeIcon` does not describe the context in which the icon is used very well. Maybe a better choice would be `getHighRuntimeHighlightingIcon`. What do you think?

Reply:
Actually, while technically it obviously is a method, semantically it’s rather some kind of constant. Also, we’ve used the naming scheme that’s used for other icons across Squeak.

I’ve renamed the method to `highRuntimeIcon`, though.